### DISCUSSION TITLE
0.10.0

### DISCUSSION THREAD

**Discussion by chalin** [google/docsy#1985] [Announcement]
## Release summary

- [Release report](https://www.docsy.dev/blog/2024/0.10.0/)
- [CHANGELOG](https://github.com/google/docsy/blob/main/CHANGELOG.md#0100)

## What's Changed

* Set NPM package version to 0.9.2-dev by @chalin in https://github.com/google/docsy/pull/1870
* Address GA.ID param deprecation warning by @chalin in https://github.com/google/docsy/pull/1879
* Upgrade to Hugo 0.123.x by @chalin in https://github.com/google/docsy/pull/1877
* UG: normalize PSC links into local blog page by @chalin in https://github.com/google/docsy/pull/1884
* UG: fix links to modules migration page by @chalin in https://github.com/google/docsy/pull/1883
* Mermaid diagram with diagram specific settings: fix 'uncaught TypeError' by @deining in https://github.com/google/docsy/pull/1885
* Bump jquery to latest released version 3.7.1 by @deining in https://github.com/google/docsy/pull/1886
* Upgrade to Bootstrap 5.3.3 by @deining in https://github.com/google/docsy/pull/1563
* Update Hugo to 0.123.8, and prettier by @chalin in https://github.com/google/docsy/pull/1891
* Update es.toml with feedback translations by @Marzal in https://github.com/google/docsy/pull/1888
* Adding Parquet Website by @vinooganesh in https://github.com/google/docsy/pull/1887
* [BS5.3] Replace text-muted by text-body-secondary by @chalin in https://github.com/google/docsy/pull/1892
* Address .Site.DisqusShortname param deprecation warning by @deining in https://github.com/google/docsy/pull/1895
* Support for adding and modifying theme colors by @chalin in https://github.com/google/docsy/pull/1893
* Update Hugo to 0.124.1 by @chalin in https://github.com/google/docsy/pull/1905
* Delete UG `layouts/sitemap.xml` by @chalin in https://github.com/google/docsy/pull/1898
* [BS5.3] Replace `.navbar-dark` by data attribute and other style adjustments by @chalin in https://github.com/google/docsy/pull/1896
* Fixed missing output redirect in Windows command line example by @dylanbeattie in https://github.com/google/docsy/pull/1903
* Style adjustments in prep for dark mode support by @chalin in https://github.com/google/docsy/pull/1908
* Color-mode / dark-mode support by @chalin in https://github.com/google/docsy/pull/1909
* [Dark-mode] Tweak of param, file and resource names by @chalin in https://github.com/google/docsy/pull/1913
* [Dark-mode] Fixes for mobile by @chalin in https://github.com/google/docsy/pull/1917
* [Dark-mode] Use default BS nav-item padding by @chalin in https://github.com/google/docsy/pull/1919
* [Dark-mode] Card shortcode style fixes by @chalin in https://github.com/google/docsy/pull/1922
* [Dark-mode] Tabbed panes fix by @chalin in https://github.com/google/docsy/pull/1920
* Adjust pageinfo bg to be compatible with dark-mode by @chalin in https://github.com/google/docsy/pull/1915
* Drop deprecated shortcode 'card-code' by @deining in https://github.com/google/docsy/pull/1923
* Markdown-link-check: ignore valid URL hash in CHANGELOG by @chalin in https://github.com/google/docsy/pull/1924
* [Dark-mode] Firefox search text color fix by @chalin in https://github.com/google/docsy/pull/1918
* Bump FontAwesome to latest released version 6.5.2 by @deining in https://github.com/google/docsy/pull/1925
* Google analytics: drop deprecated async template by @deining in https://github.com/google/docsy/pull/1931
* Upgrade Hugo to 0.125.2 by @chalin in https://github.com/google/docsy/pull/1939
* Bump algolia docsearch to latest version by @deining in https://github.com/google/docsy/pull/1933
* Fix use of deprecated param .Site.Params.DisqusShortname by @deining in https://github.com/google/docsy/pull/1934
* Bump KaTeX to latest released version by @deining in https://github.com/google/docsy/pull/1940
* [UG] Explain how to enable the light/dark-mode menu by @chalin in https://github.com/google/docsy/pull/1914
* Fix section index partial NPE by @tobiaskohlbau in https://github.com/google/docsy/pull/1890
* Drop RSS layout in favor of Hugo's default by @chalin in https://github.com/google/docsy/pull/1948
* section-index.html code cleanup by @chalin in https://github.com/google/docsy/pull/1947
* Upgrade to Hugo 0.125.4 by @chalin in https://github.com/google/docsy/pull/1950
* Docsy 0.10.0 release report by @chalin in https://github.com/google/docsy/pull/1945
* [Dark mode] Left-nav and taxonomy color fixes by @chalin in https://github.com/google/docsy/pull/1952
* Use official hugo-extended NPM package + UG tweak by @chalin in https://github.com/google/docsy/pull/1956
* Bump Persian fonts to latest version by @deining in https://github.com/google/docsy/pull/1955
* copyright.html: process `.Site.Copyright` as HTML by @chalin in https://github.com/google/docsy/pull/1954
* [UG] Syntax highlighting styles for both light and dark modes by @chalin in https://github.com/google/docsy/pull/1958
* [Dark mode] Factor out code-dark styles for reuse by @chalin in https://github.com/google/docsy/pull/1959
* Mark 0.10.0 release report as draft by @chalin in https://github.com/google/docsy/pull/1960
* Format mermaid.html by @chalin in https://github.com/google/docsy/pull/1963
* Set Mermaid theme based on light/dark mode by @chalin in https://github.com/google/docsy/pull/1964
* Mermaid: stick to default when not dark mode by @chalin in https://github.com/google/docsy/pull/1965
* Dark-mode override for `.td-box--white` by @chalin in https://github.com/google/docsy/pull/1966
* Restrict `.td-box--white` style changes to dark mode by @chalin in https://github.com/google/docsy/pull/1968
* Format lookandfeel.md - no content changes by @chalin in https://github.com/google/docsy/pull/1969
* [Dark mode] Fix blocks/lead down arrow color by @chalin in https://github.com/google/docsy/pull/1971
* Rename chroma styles by @chalin in https://github.com/google/docsy/pull/1974
* [CI] Add scripts to generate and check Chroma styles by @chalin in https://github.com/google/docsy/pull/1976
* [CI] Setup fix:format for the UG by @chalin in https://github.com/google/docsy/pull/1977
* [UG] Document light/dark mode config by @chalin in https://github.com/google/docsy/pull/1978
* Contribution guidelines: document use of Prettier by @chalin in https://github.com/google/docsy/pull/1980
* [Dark mode] blockquote and click-to-copy color fixes by @chalin in https://github.com/google/docsy/pull/1981
* [CI] use userguide/.prettierignore to specify files to ignore by @chalin in https://github.com/google/docsy/pull/1982
* Fix warning about LaTeX-incompatible input by @deining in https://github.com/google/docsy/pull/1984
* Release 0.10.0 prep by @chalin in https://github.com/google/docsy/pull/1983

## New Contributors
* @vinooganesh made their first contribution in https://github.com/google/docsy/pull/1887
* @dylanbeattie made their first contribution in https://github.com/google/docsy/pull/1903

**Full Changelog**: https://github.com/google/docsy/compare/v0.9.1...v0.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/google/docsy/releases/tag/v0.10.0'>0.10.0</a>.</em>

**Comment by chalin** [comment]
If you'd like to see the light/dark-mode menu live in production, hop over to https://opentelemetry.io.

**Comment by chalin** [comment]
@chamiz - the release has been published. Let me know how the upgrade goes the Selenium site.

**Reply by chamiz** [reply]
Thank you @chalin. I made the pull request.

**Reply by chamiz** [reply]
Thank you. I'm working the pull request

**Reply by chalin** [reply]
✅ I see that [selenium.dev](https://www.selenium.dev/) is up and running with Hugo 0.125.4 and Docsy 0.10.0 (https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1702)! 🎉 👍🏻

**Reply by chamiz** [reply]
Hi @chalin ,

Yes. My PR merged. 
All good now.  That you for your support.
